Assets/MixedRealityToolkit/Inspectors/Utilities/MixedRealityInspectorUtility.cs

```diff
@@ -22,19 +22,19 @@ public static class MixedRealityInspectorUtility
         /// Check and make sure we have a Mixed Reality Toolkit and an active profile.
         /// </summary>
         /// <returns>True if the Mixed Reality Toolkit is properly initialized.</returns>
-        public static bool CheckMixedRealityConfigured(bool showHelpBox = true)
+        public static bool CheckMixedRealityConfigured(bool renderEditorElements = true, bool showCreateButton = false)
```

Comment:
Do you know if we plan on calling this with renderEditorElements = false ever? I know that prior to this change it looks like it was used to check for the MRTK object without rendering the help box in certain cases - with this change it looks like all callers I can see within the context of this PR had their false-passing instances removed.

Reply:
It often gets called by inspectors in OnEnable when gui stuff isn't getting rendered.